framework/packages/abstract-interface/src/migrate.rs

```diff
@@ -162,28 +285,47 @@ impl<Chain: CwEnv> AbstractIbc<Chain> {
         Ok(client.is_some() || host.is_some())
     }
 
-    /// Migrate the ibc based on version changes. If the registered contracts have the right version, we don't migrate them
-    pub fn migrate_if_version_changed(&self) -> Result<bool, crate::AbstractInterfaceError> {
-        let mut has_migrated = false;
-
-        if ::ibc_client::contract::CONTRACT_VERSION != contract_version(&self.client)?.version {
-            let migration_result = self
-                .client
-                .upload_and_migrate_if_needed(&ibc_client::MigrateMsg {})?;
-            if migration_result.is_some() {
-                has_migrated = true;
-            }
+    /// Deploys or Migrates IBC Client and IBC Host
+    /// - If no version changes - don't do anything
+    /// - If version change is breaking - new version is deployed
+    /// - If version change is non-breaking - ibc contracts migrated instead
+    pub fn deploy_or_migrate_if_version_changed(
```

Comment:
if this is only ibc contracts then the fn name should reflect that. 

`upgrade_ibc_contracts` or similar. 

We should have a clear naming scheme for all the upgrade/migrate/deploy functions.

Reply:
> if this is only ibc contracts then the fn name should reflect that.
> 
> `upgrade_ibc_contracts` or similar.
> 
> We should have a clear naming scheme for all the upgrade/migrate/deploy functions.

I don't get it, It's under `AbstractIbc`, you can't skip **.ibc** .deploy_or_migrate_if_version_changed() when you use it